force-app/main/default/lwc/propertyTile/propertyTile.css

```diff
@@ -8,25 +8,33 @@
 
 .lower-third {
     position: absolute;
+    /* @sldsValidatorIgnore */
     bottom: 0;
+    /* @sldsValidatorIgnore */
     left: 0;
+    /* @sldsValidatorIgnore */
     right: 0;
     color: #fff;
     background-color: rgba(0, 0, 0, 0.3);
-    padding: 6px 8px;
+    padding: 6px var(--lwc-varSpacingXSmall, 0.5rem);
 }
 
 .lower-third > p {
+    /* @sldsValidatorIgnore */
     padding: 0;
+    /* @sldsValidatorIgnore */
     margin: 0;
 }
 
 .lower-third h1 {
+    /* @sldsValidatorIgnore */
     font-size: 16px !important;
```

Comment:
Let's finish with this PR and I'll revisit this bit of CSS in a separate PR in order to get rid of `!important`. I didn't know it was there and it's really a bad practice.